client/assets/js/main.js

```diff
@@ -1,74 +1,85 @@
-  });
+  };
 
-  $('.signUp').on('click',function(){ //clicking the sign up button at the landing page
-    $('.greeting').fadeOut();
-    $('.formSign').fadeIn();
-  });
+  document.querySelector('.signUp').onclick = function(){ //clicking the sign up button at the landing page
+    greeting.classList.add('fade-out');
+    formSign.classList.add('fade-in');
+    sign.classList.remove('fade-out');
+    sign.classList.remove('ghostOut');
+  };
 
-  $('.logIn').on('click',function(){//clicking the log in button at the landing page
-    $('.greeting').fadeOut();
-    $('.formLog').fadeIn()
-  });
+  document.querySelector('.logIn').onclick = function(){//clicking the log in button at the landing page
+    greeting.classList.add('fade-out');
+    formLog.classList.add('fade-in');
+  };
 
-  $('.cancel').on('click',function(){ // clicking the cancel on the sign up and log in forms
-    $(this).parent().parent().addClass('ghostOut').fadeOut(2500)
-    setTimeout(function(){
-        $('.formSign, .formLog').removeClass('ghostOut')
-    },3100)
-    $('.greeting').fadeIn();
-  });
+  document.querySelector('.cancel').onclick = function (event) {
+    const parent = event.target.parentNode;
+    parent.classList.add('ghostOut');
+    parent.classList.add('fade-out');
+      setTimeout(function(){
+          (formSign).classList.remove('ghostOut');
+          formSign.classList.remove('fade-in');
+          (formLog).classList.remove('ghostOut')
+          }, 1000)
+      greeting.classList.add('fade-in');
+      greeting.classList.remove('fade-out');
+  }
 
-  $('.btnLog').on('click',function(){ //clicking the log in button on the log in form
-    $(this).parent().parent().addClass('ghostOut').fadeOut(2500)
-    setTimeout(function(){
-        $('.formLog').removeClass('ghostOut')
-    },3100);
-    $('.open').fadeIn(4000).addClass('appear');
-    $('.greeting').fadeIn();
-    $('.signUp').fadeOut();
-    $('.logIn').fadeOut();
-    $('.greeting h1>span').text($('.log div input').val());
-  });
+  document.querySelector('.btnLog').onclick = function(event) {
+    const parent = event.target.parentNode;
+    parent.classList.add('ghostOut').classList.add('fade-out');
+      setTimeout(function(){
+        formLog.classList.remove('ghostOut')
+        },2500);
+      open.classList.add('fade-in, appear');
+      greeting.classList.add('fade-in');
+      signUp.classList.add('fade-out');
+      logIn.classList.add('fade-out');
+      document.querySelector('.greeting h1>span').text(document.querySelector('.username').value);
+  }
 
-  $('.btnSign').on('click',function(){ //clicking the log in button on the sign up form
-    $(this).parent().parent().addClass('ghostOut').fadeOut(2500)
-    setTimeout(function(){
-      $('.formSign').removeClass('ghostOut')
-      },3100);
-    $('.greeting').fadeIn();
-    $('.signUp').fadeOut();
-  });
+  document.querySelector('.btnSign').onclick = function(event) {
+    const parent = event.target.parentNode;
```

Comment:
when you press on sign up nothing appears. I think it's because of the fadeOut animation you attach to to it.